Makefile

```diff
@@ -29,43 +29,43 @@ $(THRIFT_BUILDDIR)/tests/integration/test.thrift_buildstamp: tests/integration/t
 	rm -f tests/integration/test_thrift/TestService-remote
 	touch $@
 
+.PHONY: venv
+venv:
+	poetry install --all-extras
+
 .PHONY: docs
 docs:
-	sphinx-build -M html docs/ build/
+	poetry run sphinx-build -M html docs/ build/
 
 .PHONY: doctest
 doctest:
-	sphinx-build -M doctest docs/ build/
+	poetry run sphinx-build -M doctest docs/ build/
 
 .PHONY: linkcheck
 linkcheck:
-	sphinx-build -M linkcheck docs/ build/
+	poetry run sphinx-build -M linkcheck docs/ build/
 
 .PHONY: test
 test: doctest
 	# Some files use gevent to monkey patch stdlib functions. This causes problems
 	# if it happens after importing the sequential versions of some of these. Thus
 	# we need to do it as early as possible.
-	python -m gevent.monkey --module pytest -v tests/
+	poetry run python -m gevent.monkey --module pytest -v tests/
 
 .PHONY: fmt
 fmt:
-	@if [ ! -f /baseplate-py-dev-docker-image ]; then \
-		echo "Please use the docker-compose environment for consistency. See https://github.com/reddit/baseplate.py/blob/develop/CONTRIBUTING.md."; \
-		exit 1; \
-	fi
-	$(REORDER_PYTHON_IMPORTS) --exit-zero-even-if-changed $(PYTHON_SOURCE)
-	black baseplate/ tests/
-	$(REORDER_PYTHON_IMPORTS) --application-directories /tmp --exit-zero-even-if-changed $(PYTHON_EXAMPLES)
-	black docs/  # separate so it uses its own pyproject.toml
+	poetry run $(REORDER_PYTHON_IMPORTS) --exit-zero-even-if-changed $(PYTHON_SOURCE)
```

Comment:
With `reorder-python-imports 3.12.0`, I get 
```
reorder-python-imports: error: unrecognized arguments: --separate-from-import --separate-relative
```

Removal of those arguments from the Makefile line 1 allows it to run for me.

Reply:
Hmm, how did you end up with that version of reorder-python-imports? I also ran into that issue initially but I pinned it back down to the version we're using prior to this branch since I was trying to avoid making additional changes:
https://github.com/reddit/baseplate.py/blob/81544dee76469a509bbf76b751871b20cc768426/pyproject.toml#L75

Did you install dependencies with Poetry? What does `poetry show reorder-python-imports` produce?

Reply:
Just did a `pipx install reorder-python-imports`

```
$ poetry show reorder-python-imports
 name         : reorder-python-imports
 version      : 2.4.0
 description  : Tool for reordering python imports

dependencies
 - aspy.refactor-imports >=2.1.0
```